packages/jest-matchers/src/index.js

```diff
@@ -35,7 +36,7 @@ if (!global[GLOBAL_STATE]) {
   Object.defineProperty(
     global,
     GLOBAL_STATE,
-    {value: {matchers: Object.create(null), state: {suppressedErrors: []}}},
+    {value: {matchers: Object.create(null), state: {expectedCounts: {}, passedExpectations: [], suppressedErrors: []}}},
```

Comment:
why does `expectedCounts` need to be an object?
can we make it a single property with either a number or `null`?